internal/cmd/scraper/scraper.go

```diff
@@ -301,6 +300,11 @@ func Run(cfg *Config) error {
 				HarvesterOpts:                 harvesterOpts,
 				DeltaExpirationAge:            cfg.TelemetryEmitterDeltaExpirationAge,
 				DeltaExpirationCheckInternval: cfg.TelemetryEmitterDeltaExpirationCheckInterval,
+				BoundedHarvesterCfg: integration.BoundedHarvesterCfg{
+					HarvestPeriod:     hTime,
+					MinReportInterval: hTime / 10,
```

Comment:
should we set this with a hardcoded value to "save" the backend? Es 50ms?

Reply:
Yes, the current `/10` is very temporary. I have yet to do some load testing to find out what is a "reasonable" default. 100ms worked for our 1000 targets, but might need further refinement.

In any case, the worst case scenario for this being set "too high" is just that memory will start to increase. I wouldn't concern too much if 3000 targets caused ~1-2GiB of ram to be used, still seems reasonable to me.

Reply:
All these parameters are now settable through the config.